### PR TITLE
solved js1-prob10

### PR DESCRIPTION
**Pull request by fmtlic**
please review

### CODE REVIEW

File: curriculum/js1/10.js [hunk 1]
```diff
@@ -7,7 +7,9 @@
  * @param {function} c
  */
 
-const solution = (a, b, c) => {
+const solution = (a, b, C) => {
+  setTimeout (C, a*1000)
+  setTimeout (C, (a+b)*1000)
```
Comment (danielthalhuber) [inline curriculum/js1/10.js]: Lines 11 & 12:
* Please remove this space after `setTimeout`. `setTimeout` is a method/function. Common style guides will specify no space between the a function name and the opening bracket.

* Please add semicolons. JavaScript will let you get away with this, but most people will expect to see semicolons after these function calls.

Comment (fmtlic) [reply curriculum/js1/10.js]: Hi Daniel,

Thanks for the comments. I made changes and pushed the branch again to github just now.

Can you review again? Also, do I have to make a new merge request?

Thanks!

File: curriculum/js1/10.js [hunk 2]
```diff
@@ -7,7 +7,9 @@
  * @param {function} c
  */
 
-const solution = (a, b, c) => {
+const solution = (a, b, C) => {
+  setTimeout(C, a*1000);
+  setTimeout(C, (a+b)*1000);
```
Comment (danielthalhuber) [inline curriculum/js1/10.js:12]: I will approve this. But I have a comment on the comment. The comment on line 7 says that the third parameter is `c`, but the parameter is actually named `C`. There's nothing illegal about using an uppercase variable name, but the convention is to start the name with a lowercase letter. Generally, names that start with an uppercase letter are class-names or constructors.

**Review by danielthalhuber — APPROVED** [review]

File: curriculum/js1/10.js (hunk 2 above)
Comment (fmtlic) [reply curriculum/js1/10.js:12]: Got it. Thanks!

**Comment by fmtlic** [comment]
Got it. Thanks!

Blessings,

Tae Young David Kim

Cell: +1-714-332-7103

**Pull request merged**